cli/__init__.py

```diff
@@ -1,6 +1,14 @@
 import os
+import shutil
+import subprocess
 import sys
 
+from packaging.version import Version, parse
+
+KNOWN_PLUGINS_URL = (
+    'https://raw.githubusercontent.com/jina-ai/jina/master/cli/known_plugins.json'
+)
```

Comment:
I believe we could make this accessible through the `https://api.jina.ai/` API instead, but I don't know how to set that up